src/Plugin/CKEditorPlugin/OembedEntities.php

```diff
@@ -71,9 +64,8 @@ public function __construct(
     EntityTypeManagerInterface $entityTypeManager,
     ModuleExtensionList $moduleExtensionList = NULL
   ) {
-    parent::__construct($configuration, $plugin_id, $plugin_definition, $embed_button_query);
+    parent::__construct($configuration, $plugin_id, $plugin_definition, $entityTypeManager, \Drupal::service('csrf_token'));
```

Comment:
Why we didn't do an injection of `csrf_token`?

Reply:
It's part of the temporary changes based on embed PR.